25 calculate liquidation profit improvements

Bitcoinera opened the pull request:
Close #25 

Have been trying already for couple of days to get to see the logs from the two "profits" :sweat_smile: 
The one from `calculateLiqProfit` and the one from ` parseFloat(balAfter) - parseFloat(balBefore)`... but when I'm about to see them, I run out of positions. Somehow I never get to see these logs. Will keep trying

Bitcoinera committed:
fix returned liquidation profit

Bitcoinera committed:
homogenize launch scripts

Bitcoinera committed:
hyper important fixes

Bitcoinera committed:
this will give conflicts later but...

Bitcoinera committed:
Merge branch '27-fix-insufficient-funds-to-pay-for-pending-and-new-transaction' of https://github.com/DistributedCollective/Sovryn-Node into 25-calculate-liquidation-profit-improvements

Bitcoinera committed:
fix undefined in swapBackAfterLiquidation

Bitcoinera committed:
add tradeCtrl script to util

Bitcoinera committed:
fixes for tradeCtrl script

Bitcoinera committed:
add bulk positions creation

Bitcoinera commented:
Taking more time than expected as I need to have liquidatable positions for this

jamiemadrox779 commented:
> Taking more time than expected as I need to have liquidatable positions for this

Why not taking the logs from successfully liquidated positions as template?

Bitcoinera committed:
remove worse way to calculate liquidation profit

Bitcoinera committed:
hyper important fix

Bitcoinera committed:
shift to short positions and higher leverage

Bitcoinera committed:
Merge branch 'development' of https://github.com/DistributedCollective/Sovryn-Node into 25-calculate-liquidation-profit-improvements

Bitcoinera commented:
Ready to go. The way I introduced to calculate liquidity was more unreliable than the old way (it was needed to do a conversion and also depended on the priceFeed changes...). The old way is much more precise. 

I also fixed a bug with the numberOfHops. This was the reason why profit was not being stored. It was always doing return and skipping the rest of the function (line 190 in addLiqLog function).

Bitcoinera committed:
avoid using hardcoded contract address

jamiemadrox779 committed:
Merge branch 'development' into 25-calculate-liquidation-profit-improvements
